examples/gno.land/p/demo/diff/diff.gno

```diff
@@ -0,0 +1,151 @@
+package diff
+
+import (
+	"strings"
+)
+
+type EditType int
+
+const (
+	EditKeep EditType = iota
+	EditInsert
+	EditDelete
+)
+
+type Edit struct {
+	Type EditType
+	Char rune
+}
+
+func MyersDiff(old, new string) []Edit {
```

Comment:
```suggestion
func MyersDiff(old, new []byte) []Edit {
```

And probably other changes induced by using `[]byte` instead of `string`.

Reply:
I skipped this content for the same reason as above